README.md

```diff
@@ -8,78 +8,29 @@ Setup
 ### Prerequisites
 
 Working git, ruby or jruby, and rubgems, wget
-Gems: capistrano, capistrano-ext, ruby-debug (oddly, haml too, when
-deploying …)
-TODO: understand why running capistrano requires haml…
-
-### Install
-
-The latest setup scripts have been tested with 1.4.0RC1 of JRuby, you
-will need a running mysql
-server and a mysql username/password with enough permissions to create
-and drop the
-database you intend to use with RITES.
-
-#### RVM, Bundler, & Gem Dependencies
-
--   [Maintaining gem dependencies](doc/gem-dependencies.textile)
-
-#### Updating to a newer codebase.
-
--   Example of [updating an older instance to to newer
-    code](doc/updating_an_older_instance.md)
 
 #### Core Extensions
 
 -   [Extensions to core classes applied at application
     startup](doc/core-extensions.textile)
 
-#### Setting up a Hudson CI project
-
--   [Setting up a Hudson CI project](doc/hudson-ci.textile)
-
-#### GIT
-
-In the example below we use the xproject theme and also as the name of
-the directory and
-the prefix of the names for the databases that will be created in mysql.
-
-If you are a committer in this repo:
-
-    git clone git@github.com:concord-consortium/rigse.git xproject
+#### Simple Getting Started
 
-If you are not a committer:
-
-    git clone git://github.com/concord-consortium/rigse.git xproject
-
-Change to the project directory:
-
-    cd xproject
-
-Setup rvm to use Ruby 1.9.3-p125 and a gemset named xproject
-(newer versions of 1.9.3 ought to work as well)
-
-    rvm use 1.9.3-p125
-    rvm gemset create xproject
-
-Make an `.rvmrc` file so rvm will use this ruby gemset combo
-automatically when you change to this directory:
-
-    echo "rvm use 1.9.3-p125@xproject" > .rvmrc
-
-Test out the `.rvmrc` file and start using the gemset
-
-    cd ..;cd xproject
+    bundle install
+    cp config/database.sample.yml config/database.yml (need to fix the mysql password and/or user)
+    cp config/settings.sample.yml config/settings.yml
+    cp config/app_environment_variables.sample.rb config/app_environment_variables.rb
+    cp config/sis_import_data.sample.yml config/sis_import_data.yml
```

Comment:
We are removing sis_import in https://github.com/concord-consortium/rigse/pull/92  -- so we can probably delete this copy.